tests/ui/components/results/paginationcomponent.js

```diff
@@ -144,3 +147,48 @@ describe('rendering the page numbers', () => {
     expect(paginate).toHaveBeenCalledWith(2, 1, 5);
   });
 });
+
+describe('properly interacts with storage', () => {
+  let defaultConfig;
+  beforeEach(() => {
+    const bodyEl = DOM.query('body');
+    DOM.empty(bodyEl);
+    DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+
+    defaultConfig = {
+      container: '#test-component',
+      verticalKey: 'verticalKey'
+    };
+  });
+
+  it('the limit property is set to the global storage search-config.limit after being created', () => {
+    const component = COMPONENT_MANAGER.create('Pagination', {
+      ...defaultConfig
+    });
+
+    const limit = component._limit;
```

Comment:
could we test this through the ui instead of the variable value?

Reply:
This value doesn't show up in the UI so i'm not sure if testing it in the UI makes sense

Reply:
would it be possible to test that this is working by mocking out the results count and search offset?

Reply:
I could test it indirectly by mocking resultsCount and searchOffset and checking if the nextPageButton is shown. Is that what you mean? Whether or not nextPageButton is shown depends on resultsCount, searchOffset, and limit

Reply:
it looks like limit is used to calculate stuff like the current and max page number, so maybe testing for those values in the ui? I think it would be just a little more robust

Reply:
sure